modules/lambda-example/deploy.tf

```diff
@@ -0,0 +1,32 @@
+module "lambda-with-container" {
+  source      = "../lambda-with-container"
+  codebase    = path.module
```

Comment:
Is this just going to zip up everything in this directory?

Reply:
I guess my question is will stuff listed in the dockerignore file be excluded from what gets shipped or does the archive zip end up bringing it in?

Reply:
We use the sha256 of the archive to decide if any changes require that the image is rebuilt. The archive itself doesn't get used outside of that.

Reply:
Cool, means you'd have an image rebuild even if ignored file (tests etc) got modified right? But I think that's probably fine.

Reply:
At the moment, yes. That could be improved, but its probably fine for the moment. One thing I did want to introduce was layer caching to speed up the build time. In testing build time was around 2 minutes without this but could be improved to around 30 seconds per docker image build.